x/fungible/keeper/msg_server_deploy_fungible_coin_zrc20.go

```diff
@@ -37,7 +37,7 @@ func (k msgServer) DeployFungibleCoinZRC20(goCtx context.Context, msg *types.Msg
 	var address common.Address
 	var err error
 
-	if msg.Creator != k.observerKeeper.GetParams(ctx).GetAdminPolicyAccount(zetaObserverTypes.Policy_Type_deploy_fungible_coin) {
+	if msg.Creator != k.observerKeeper.GetParams(ctx).GetAdminPolicyAccount(zetaObserverTypes.Policy_Type_group1) {
```

Comment:
For now i think this should be group2, 
as this can overwrite the gas ZRC20 contract.

Reply:
Maybe we could have group 2 for gas and group 1 for regular zrc20?

Reply:
76df01bc